src/main/java/uk/gov/ons/ctp/response/collection/exercise/validation/ValidationScheduler.java

```diff
@@ -11,17 +14,34 @@
  */
 @Component
 public class ValidationScheduler {
+  private static final String VALIDATION_GLOBAL_LOCK_NAME = "SampleValidationCollexLock";
 
   private SampleService sampleService;
+  private RedissonClient redissonClient;
 
   @Autowired
-  public ValidationScheduler(SampleService sampleService) {
+  public ValidationScheduler(SampleService sampleService, RedissonClient redissonClient) {
     this.sampleService = sampleService;
+    this.redissonClient = redissonClient;
   }
 
   /** Carry out scheduled validation according to configured fixed delay. */
   @Scheduled(fixedDelayString = "#{appConfig.schedules.validationScheduleDelayMilliSeconds}")
   public void scheduleValidation() {
-    sampleService.validateSampleUnits();
+    RLock lock = redissonClient.getFairLock(VALIDATION_GLOBAL_LOCK_NAME);
+
+    // Get a lock. Automatically unlock after a certain amount of time to prevent issues
+    // when lock holder crashes or Redis crashes causing permanent lockout
+    try {
+      if (lock.tryLock(1, TimeUnit.HOURS)) {
+        try {
+          sampleService.validateSampleUnits();
+        } finally {
+          lock.unlock();
+        }
+      }
+    } catch (InterruptedException e) {
+      // Ignored
```

Comment:
Should probably at least log

Reply:
this happens only in the rare event that the thread is being killed by a shutdown. I'm not sure we'd have enough time to log and what would we log except to say that the thread was forcibly killed, which isn't really an error or anything.